src/diagnostics/default_diagnostics.jl

```diff
@@ -230,6 +230,19 @@ function default_diagnostics(
     return [average_func(precip_diagnostics...; output_writer, start_date)...]
 end
 
+function default_diagnostics(
+    ::Microphysics2Moment,
+    duration,
+    start_date;
+    output_writer,
+)
+    precip_diagnostics = ["husra", "hussn", "cdnc", "cdinc", "ncra", "ncsn"]
```

Comment:
We will probably want to revisit this after P3 is added. Or maybe just add the warm rain parts here?